src/db.ts

```diff
@@ -230,6 +230,13 @@ export class DB {
     this._open = false;
   }
 
+  /**
+   * Get last inserted row id
```

Comment:
Please format the comment such that it will be included in the generated documentation, and also be specific about what SQLite function this corresponds to:
```suggestion
   * DB.lastInsertRowId
   *
   * Get last inserted row id. This corresponds to
   * the SQLite function `sqlite3_last_insert_rowid`.
```

Reply:
Fixed